internal/testplanet/planet.go

```diff
@@ -472,7 +481,9 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 		config.Console.StaticDir = filepath.Join(storjRoot, "web/satellite")
 		config.Mail.TemplatePath = filepath.Join(storjRoot, "web/satellite/static/emails")
 
-		peer, err := satellite.New(log, identity, db, &config)
+		verClient := planet.NewVersionClient()
```

Comment:
Service should be created inside the peer and the version control address passed through the config.